operations/iuf/workflows/management_rollout.md

```diff
@@ -488,6 +488,13 @@ The worker canary node can be any worker node and does not have to be `ncn-w001`
       $ncn --format json | jq -r ' .id+" "+.desiredConfig+" status="+.configurationStatus'; done
     ```
 
+1. (`ncn-m001#`) If this is an upgrade to CSM 1.6.0, it is required to restart the 'goss-servers' service on all worker nodes as some Goss tests may not be included due to a timing issue. This issue is fixed in CSM 1.6.1.
+
+    ```bash
+    worker_nodes=$(grep -oP "(ncn-w\d+)" /etc/hosts | sort -u | tr -t '\n' ',')
```

Comment:
Even though `pdsh` doesn't seem to mind it, please strip off the trailing comma:
```
 worker_nodes=${worker_nodes%,}
```

Reply:
I have included this in my new changes, thanks.